client/src/components/desktop/ResourcesFilter.js

```diff
@@ -31,17 +34,20 @@ function ResourcesFilter(props) {
         case 'Location':
           setLocation(value);
           break;
+        case 'Sort By':
+          setSort(value);
+          break;
         default:
       }
     },
-    [setCost, setLanguage, setLocation],
+    [setCost, setLanguage, setLocation, setSort],
   );
 
   const radio = useCallback(
-    (filterName, filterOptions, value) => {
+    (filterName, filterOptions, value, isSort) => {
       return (
-        <div className="radio-container">
-          <h4 className="title">{filterName}</h4>
+        <div className={isSort ? 'radio-container-sort' : 'radio-container'}>
+          {isSort ? null : <h4 className="title">{filterName}</h4>}
```

Comment:
What you can do is use [conditional rendering](https://reactjs.org/docs/conditional-rendering.html)

`{!isSort && <h4 className="title">{filterName}</h4>}`